src/components/InvertedFlatList/index.js

```diff
@@ -18,34 +18,30 @@ const propTypes = {
 
 // This is adapted from https://codesandbox.io/s/react-native-dsyse
 // It's a HACK alert since FlatList has inverted scrolling on web
-class InvertedFlatList extends React.Component {
-    constructor(props) {
-        super(props);
+function InvertedFlatList(props) {
+    const {innerRef, contentContainerStyle} = props;
+    let list;
 
-        this.list = undefined;
-    }
-
-    componentDidMount() {
-        if (!_.isFunction(this.props.innerRef)) {
+    useEffect(() => {
+        if (!_.isFunction(innerRef)) {
             // eslint-disable-next-line no-param-reassign
-            this.props.innerRef.current = this.list;
+            innerRef.current = list;
         } else {
-            this.props.innerRef(this.list);
+            innerRef(list);
         }
-    }
-
-    render() {
-        return (
-            <BaseInvertedFlatList
-                // eslint-disable-next-line react/jsx-props-no-spreading
-                {...this.props}
-                inverted
-                ref={(el) => (this.list = el)}
-                shouldMeasureItems
-                contentContainerStyle={StyleSheet.compose(this.props.contentContainerStyle, styles.justifyContentEnd)}
-            />
-        );
-    }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
```

Comment:
```suggestion
    }, [innerRef, list]);
```

I know there was `componentDidMount` before, but there is no need to transform it 1:1 with empty deps array and eslint comment.